app.py

```diff
@@ -84,37 +84,49 @@ async def command(ack, body, respond, client, logger):
     datestring = today.strftime("%Y-%m-%d")
     user_id = body.get("user_id")
 
-    channel_id, channel_name = get_channel_id_and_name(body, logger)
+    # Figure out where user sent slashcommand from to set current channel id and name
+    is_direct_message = body.get("channel_name") == 'directmessage'
+    current_channel_id = user_id if is_direct_message else body.get("channel_id")
+    current_channel_name = "Me" if is_direct_message else body.get("channel_id")
     
+    # The channel where user submitted the slashcommand 
+    current_channel_option = {
+        "text": {
+            "type": "plain_text",
+            "text": "Current Channel"
+        },
+        "value": current_channel_id
+    }
```

Comment:
This is the new, smart default option we discussed last week in Slack so if there is no CHANNEL set, then the channel the user sent slashcommand from is set as the default.